velox/functions/sparksql/specialforms/SparkCastHooks.cpp

```diff
@@ -69,6 +69,12 @@ Expected<Timestamp> SparkCastHooks::castIntToTimestamp(int64_t seconds) const {
   return castNumberToTimestamp(seconds);
 }
 
+Expected<int64_t> SparkCastHooks::castTimestampToInt(
+    Timestamp timestamp) const {
+  return std::floor(
+      timestamp.toMicros() / (Timestamp::kMicrosecondsInSecond * 1.0));
```

Comment:
> Timestamp::kMicrosecondsInSecond * 1.0

nit: can we use `static_cast<double>` for this conversion to improve readability?